packages/plugin-core/src/views/NativeTreeView.ts

```diff
@@ -111,6 +118,15 @@ export class NativeTreeView implements Disposable {
       );
       if (activeNote !== undefined) {
         treeView.reveal(activeNote);
```

Comment:
can we just reveal and set `focus` to false? see https://code.visualstudio.com/api/references/vscode-api#TreeView%3CT%3E

Reply:
the options for `.reveal` pertains to the tree _item_ that is being revealed.

This is irrelevant now (I removed it), as we are restricting the forced focus to only once during the tutorial workspace activation. This was a workaround to give back focus to the already open text editor for just-activated existing workspaces

Reply:
This does mean that the focus _will_ be on the tree view for folks who first open a tutorial workspace.
If we want to focus to be on the tutorial note, we'll have to add this back in (ideally not here but during workspace activation.)